### PR TITLE
Fix icon for API < 26

### PR DESCRIPTION
**Pull request by Unpublished**

### CODE REVIEW

**Comment by Unpublished** [comment]
This shouldn't break API >= 26 but didn't verify yet.

**Comment by menny** [comment]
this is crazy... thanks for the fix.
Can you completely delete that folder? `app/src/main/res/mipmap-anydpi`

File: app/src/main/res/values-v26/ic_launcher_background.xml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <color name="ic_launcher_background">#FFFFFF</color>
```
Comment (menny) [inline app/src/main/res/values-v26/ic_launcher_background.xml]:
```suggestion
  <color name="ic_launcher_background">#D4D4D4</color>
```
The file you deleted (`app/src/main/res/values/ic_launcher_background.xml`) had a color value of #D4D4D4 and not #FFFFFF.

**Comment by menny** [comment]
Thank you

**Pull request merged**